### PR TITLE
Fixing some undefined variables bugs #1161 and #1162

### PR DESCRIPTION
**Pull request by panchenko**
Fixes as I understand it should be.

The `opt_internal_events` identifier is used only once, so IMHO that line can be removed.

### CODE REVIEW

File: src/coffee/directives/api/models/parent/rectangle-parent-model.coffee [hunk 1]
```diff
@@ -22,8 +22,8 @@ angular.module('uiGmapgoogle-maps.directives.api.models.parent')
         else if scope.bounds.getNorthEast? and scope.bounds.getSouthWest? #google format
             bounds = scope.bounds
         else
-          if bound?
-            $log.error "Invalid bounds for newValue: #{JSON.stringify scope.bounds}" #note if bouds is recursive this could crash
+          if scope.bounds?
```
Comment (nmccready) [inline src/coffee/directives/api/models/parent/rectangle-parent-model.coffee]: this probably should just be `bounds?` which is created above at `bounds = @convertBoundPoints(scope.bounds)`

**Comment by nmccready** [comment]
Looks good besides comments.

File: src/coffee/directives/api/models/parent/rectangle-parent-model.coffee (hunk 1 above)
Comment (panchenko) [inline src/coffee/directives/api/models/parent/rectangle-parent-model.coffee]: I am not sure, if that line is executed - the last `else` is never reached, also all other conditions here check `scope.bounds`.

Comment (nmccready) [inline src/coffee/directives/api/models/parent/rectangle-parent-model.coffee]: Ahh, your right my bad.

**Pull request merged**